images/dind/docker.service

```diff
@@ -1,15 +0,0 @@
-[Unit]
-Description=Docker-in-Docker (dind)
-After=network.target
-
-[Service]
-Type=notify
-EnvironmentFile=-/etc/sysconfig/docker-network
```

Comment:
it looks like this line (`EnvironmentFile=-/etc/sysconfig/docker-network`) gets dropped in the new dind.conf file?